artemis-core-client/src/main/java/org/apache/activemq/artemis/core/remoting/impl/netty/TransportConstants.java

```diff
@@ -125,6 +125,10 @@
     */
    public static final String NIO_REMOTING_THREADS_PROPNAME = "nioRemotingThreads";
 
+   public static final String NETTY_WRITE_BUFFER_LOW_WATER_MARK_PROPNAME = "nettyWriteBufferLowWaterMark";
```

Comment:
You're right, I've used the Netty prefix just because that values are valid only under Netty transport, but a different name is welcome